pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -31,13 +31,13 @@ This section describes how to upgrade your Kommander Management cluster and all
   Download the Kommander charts bundle:
 
   ```bash
-  wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-kommander-charts-bundle-v2.2.0.tar.gz"
+  wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-kommander-charts-bundle-v2.2.0.tar.gz" -O - | tar -xvf -
```

Comment:
I have updated in `konvoy/2.2/...` dir as well but i found that this file is significantly different from the one in `kommander/2.2/...` dir, so i copied all the changes over. can you PTAL :pray:

Reply:
Hi @takirala, thanks for copying this over! I am 98% sure those changes are needed here too. (The MinIO attributions required us to double untar several images) @mackdubya, this change is required here too, right?